packages/clis/generator/geo-json.ts

```diff
@@ -731,6 +735,105 @@ export function convertToFootprintsGeoJson({
   };
 }
 
+export function convertToContoursGeoJson({
+  map,
+  points,
+}: {
+  map: 'usa' | 'europe';
+  points: [number, number, number][];
+}) {
+  const normalizeCoordinates = createNormalizeCoordinates(map, 4);
+  let min = Infinity;
+  let max = -Infinity;
+  for (const p of points) {
+    if (p[2] < min) {
+      min = p[2];
+    }
+    if (p[2] > max) {
+      max = p[2];
+    }
+  }
+  const levels = max - min + 1;
+
+  logger.log('calculating sector mask...');
```

Comment:
I _really_ wanted to solve the problem of seeing elevation data outside of sector bounds by making use of [constrained delaunay triangulation](https://en.wikipedia.org/wiki/Constrained_Delaunay_triangulation): `d3-tricontours` lets you specify a custom triangulation function, and it would've been great if I could just plug-in [poly2tri.js](https://github.com/r3mi/poly2tri.js).

But poly2tri.js appears to do its triangulation recursively and choked on the millions of points I gave it. And I didn't want to spend any more time on dealing with that issue, given how inaccurate the elevation data I'm using is, anyway (I'm gathering elevation data so naively and incompletely).

So simple polygon masking it is!